controllers/usersController.js

```diff
@@ -0,0 +1,33 @@
+const BaseController = require("./baseController");
+
+class UsersController extends BaseController {
+  constructor(model) {
+    super(model);
+    this.userModel = model;
```

Comment:
Because you are extending baseController and used super(model)
this.model will be equal to model base on the extension. No need to defined this.userModel = model because this.userModel = this.model